trueline.sh

```diff
@@ -445,6 +445,35 @@ _trueline_cmd_duration_segment() {
     fi
 }
 
+_trueline_distro_icon_segment() {
+    
+    set_distribution_icon() {
+        case "$(uname -a)" in
+            *ubuntu*) distro_icon=" " ;;
+            *fedora*) distro_icon=" " ;;
+            *debian*) distro_icon=" " ;;
+            *arch*) distro_icon=" " ;;
+            Linux*) distro_icon=" " ;;
+            *) distro_icon="${TRUELINE_SYMBOLS[empty_distribution]} " ;;
+        esac
+    }
+
+    # Call the set_distribution_icon function to set the distro_icon.
+    set_distribution_icon
```

Comment:
I don't know why you need a function for this? can't you simply se the distro_icon variable in the case above and avoid defining the function  (and the function call) alltogether?

Reply:
Yep you are right. We don't need a function here true.